### ISSUE TITLE
Resource Management - UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#18]
## User story
1. As a user (admin)
2. I want / need to be able to see all resources added by my company
3. So that I can add new ones, edit or delete them

## Acceptance criteria
* The resource management page should only be visible for admins
* The resource management page should provide a list of all available resources of that company with their respective properties
* The resource management page should offer a button for editing and one for deleting a resource as well as an option for adding a new resource
* The resource management page should offer a search field to quickly find certain resources
* The page should offer options to filter the displayed resources depending on certain properties, such as country, location, room and type and update the displayed list of resources accordingly

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
